src/watchers/flagcandprof.js

```diff
@@ -0,0 +1,49 @@
+const {loadConfig} = require('../functions');
+
+const config = loadConfig();
+
+class FlagsHandler {
+
+    async action({doc, dac_directory, db}) {
+        const {
+            action: {
+                account,
+                name,
+                data: {
```

Comment:
Can it happen that data will be undefined?

Reply:
I don't think so. I believe all actions will have data. Otherwise there is no point for us to track them. It's technically possible to have the data be an empty object ( eg. `{}` ). But I would still expect it to be at least defined. All the actions we track have at least one param.

Reply:
ok